cmd/migrate.go

```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"github.com/evcc-io/evcc/core/keys"
+	"github.com/evcc-io/evcc/server/db/settings"
+	"github.com/spf13/cobra"
+)
+
+// migrateCmd represents the migrate command
+var migrateCmd = &cobra.Command{
+	Use:   "migrate",
+	Short: "Migrate yaml to database (overwrites db settings)",
+	Args:  cobra.ExactArgs(0),
+	Run:   runMigrate,
+}
+
+func init() {
+	rootCmd.AddCommand(migrateCmd)
+}
+
+func runMigrate(cmd *cobra.Command, args []string) {
+	// load config
+	if err := loadConfigFile(&conf, !cmd.Flag(flagIgnoreDatabase).Changed); err != nil {
+		log.FATAL.Fatal(err)
+	}
+
+	// TODO remove yaml file
+	settings.SetInt(keys.Interval, int64(conf.Interval))
+
+	// TODO add remaining settings @naltatis (site? loadpoints? residualpower?)
```

Comment:
missing values are: `siteTitle`, `sponsorToken`, `residualPower`,  `maxGridSupplyWhileBatteryCharging`,

Reply:
Es lassen sich nur die globalen Settings migrieren, also das Token. Der Rest ist in der global Config aktuell nicht ausmodelliert. Das müssten wir sonst noch ändern. Ich schaue nochmal...